packages/react-core/src/Authenticator/hooks/useAuthenticator/__tests__/useAuthenticator.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
Should we keep the test extensions to `.test.tsx` for the sake of consistency? I don't have a preference, but imo we should use one.

Reply:
Not super opinionated about aligning/not-aligning, but i do default to _.spec_ personally. Just for awareness:
- our linting and TS configs are set to allow both conventions
- unit tests on the in-app branch use _.spec_

Reply:
I'm not opinionated either, it's just nice when you can search `.test.tsx` on vscode file search window. But anyway it's a very nit thing that I do not need to block on.